### PR TITLE
add_configurability

### PR DESCRIPTION
**Pull request by GabrielLaden-minted**
I'm adding a few things to make geerlingguy.elasticsearch more configurable:
* package version
* more default variables
* more templating

I'm going to upload this to galaxy after it's merged.

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@ dependencies:
   - geerlingguy.java
 
 galaxy_info:
-  author: geerlingguy
+  author: minted
   description: Elasticsearch for Linux.
-  company: "Midwestern Mac, LLC"
+  company: "minted"
   license: "license (BSD, MIT)"
```
Comment (patrick-minted) [inline meta/main.yml]: change the license to None

File: tasks/setup-Debian.yml [hunk 2]
```diff
@@ -11,4 +11,8 @@
     update_cache: yes
 
 - name: Install Elasticsearch.
-  apt: pkg=elasticsearch state=present
+  apt: pkg=elasticsearch={{ es_version }}
```
Comment (patrick-minted) [inline tasks/setup-Debian.yml]: can we use the yaml format?

apt:
  pkg: "elasticsearch={{ es_version }}"

File: tasks/setup-Debian.yml [hunk 3]
```diff
@@ -11,4 +11,8 @@
...
+
+- name: Add init.d file for Elasticsearch.
+  template: src=elasticsearch.j2 dest=/etc/init.d/elasticsearch mode=0755 force=yes
```
Comment (patrick-minted) [inline tasks/setup-Debian.yml]: template: 
__  src: elasticsearch.j2
__  dest: /etc/init.d/elasticsearch
__  mode: 0755 
__  force:yes

File: defaults/main.yml [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+
+# /etc/init.d/elasticsearch
+es_heap_size: 2g
+
+# /etc/elasticsearch/elasticsearch.yml
+es_cluster_name: mycluster
+es_node_name: "{{ ansible_hostname }}"
+es_http_port: 9200
+es_network_host: localhost
+es_host_list: localhost
```
Comment (patrick-minted) [inline defaults/main.yml]: this should be [localhost]

**Comment by patrick-minted** [comment]
👍 after making changes

**Comment by GabrielLaden-minted** [comment]
It occurred to me that I should also update the Readme

**Pull request merged**